tests/arp/conftest.py

```diff
@@ -50,9 +50,27 @@ def intfs_for_test(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_fro
         intf1 = ports_for_test[0]
         intf2 = ports_for_test[1]
     else:
-        # Select port index 0 & 1 two interfaces for testing
-        intf1 = ports[0]
-        intf2 = ports[1]
+        # Select first 2 ports that are admin 'up'
```

Comment:
Why would testbed yaml file include ports that are admin down? If all the ports in testbed file are admin up, we don't need the change in the test, right?

Reply:
yaml file contains the testbed topology information. Sometimes the links are not up, not necessarily because of 'admin' state. It's good to get working ports for testing to reduce flakiness.